Sources/Event Logging/TracksEventPersistenceService.h

```diff
@@ -18,4 +20,11 @@
 
 - (void)incrementRetryCountForEvents:(NSArray *)tracksEvents;
 
+/// Increments the retry count for the specified events, and offers a completion closure with
+/// error handling support.
+/// 
+- (void)incrementRetryCountForEvents:(NSArray *)tracksEvents onComplete:(nullable void(^)(NSError * _Nullable error))completion;
```

Comment:
Only used for testing right now, but this basically offers support to have what we had before (ie: when the increment was run synchronously).